packages/one-app-bundler/__tests__/webpack/loaders/__snapshots__/externals-loader.spec.js.snap

```diff
@@ -1,10 +1,36 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports['externals-loader should ignore the content and get the dependency from the root module 1'] = `
+exports[`externals-loader does not use fallback for server 1`] = `
```

Comment:
im not sure this snapshot captures what the intent is

Reply:
updated